src/settings.cc

```diff
@@ -244,11 +244,23 @@ Napi::Value getTraceSettings(const Napi::CallbackInfo& info) {
     return o;
   }
 
+  // oboe sets sample_source to -1 when it was a
+  // "continue" decision OR when it is a trigger trace.
+
+  // for a continued trace (that has metadata) oboe will set
+  // out->sample_rate = -1;
+  // out->sample_source = -1;
+  // out->token_bucket_rate = -1;
+  // out->token_bucket_capacity = -1;
+
+  // for a trigger trace (that DOES NOT have metadata) oboe will set
+  // out->sample_source = -1;
+
+  // the conditional trio represents a continued trace which has metadata.
+  have_metadata = out.sample_rate == -1 && out.sample_source == -1 && out.token_bucket_rate == -1 && out.token_bucket_capacity == -1;
```

Comment:
Noted that using this line (the oldest way to determine it is a continued trace or not) doesn't require liboboe 10.5.0 support.

Reply:
Is required for 10.4.0 and above.